### PR TITLE
add keymap: happysalada

### PR DESCRIPTION
**Pull request by happysalada**
## Description

Just adding my personal keymap

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/keymap.c [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+*/
+
+#include QMK_KEYBOARD_H
+
+#define _COLEMAK_DH  0
+#define _LEFT     1
+#define _LEFT_UP  2
+#define _RIGHT    3
+#define _RIGHT_UP 4
+#define _QWERTY   5
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/keymap.c:25]:
```suggestion
enum custom_layers {
    _COLEMAK_DH,
    _LEFT,
    _LEFT_UP,
    _RIGHT,
    _RIGHT_UP,
    _QWERTY,
};
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/keymap.c (hunk 1 above)
Comment (happysalada) [reply keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/keymap.c]: Thanks for taking the time to review!

**Review by fauxpark — APPROVED** [review]

File: keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/config.h [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/*
```
Comment (drashna) [inline keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/config.h:1]: Technically, this file isn't needed at all.  

serial is the default transport method if there is no define. And the same with master left.  So omitting this file would have the same functionality.

Comment (happysalada) [reply keyboards/handwired/dactyl_manuform/6x6/keymaps/happysalada/config.h]: Thanks!

**Review by drashna — APPROVED** [review]

**Pull request merged**